### ISSUE TITLE
CSS Demo fails color blind a11y

### ISSUE THREAD

**Issue by mjpoo** [mdn/content#39201]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/@media

### What specific section or headline is this issue about?

The Try It CSS Demo: @media

### What information was incorrect, unhelpful, or incomplete?

As a colour blind person I could not see that the NASA text changed from red to limegreen when I hover over the abbreviation in the example.

### What did you expect to see?

Use a colour other than green.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/@media`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/@media
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/@media/index.md
* Last commit: https://github.com/mdn/content/commit/b2c8dcdae36907a87d1d1b9393ca4a35ebc765d6
* Document last modified: 2025-04-12T01:47:48.000Z

</details>
